src/ansys/mapdl/core/_commands/apdl/process_controls.py

```diff
@@ -22,27 +22,324 @@
 
         Notes
         -----
-        You should consider using ``time.sleep(dtime)``
+        Causes the next read to be from a specified line on the input file. Lines may be skipped or reread.
+        The :ref:`stargo` command will `not` be executed unless it is part of a macro, user file (processed
+        by :ref:`use` ), an alternate input file (processed by :ref:`input` ), or unless it is used in a
+        batch-mode input stream. Jumping into, out of, or within a do-loop or an if-then-else construct to a
+        : ``label`` line is not allowed.  This command is valid in any processor.
+        """
+        command = f"*GO,{base}"
+        return self.run(command, **kwargs)
+
+    def wait(self, dtime: str = "", **kwargs):
+        r"""Causes a delay before the reading of the next command.
 
-        The command following the /WAIT will not be processed until the
-        specified wait time increment has elapsed.  Useful when reading from a
-        prepared input file to cause a pause, for example, after a display
-        command so that the display can be reviewed for a period of time.
-        Another "wait" feature is available via the ``*ASK`` command.
+        Mechanical APDL Command: `/WAIT <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_WAIT.html>`_
 
-        This command is valid in any processor.
+        Parameters
+        ----------
+        dtime : str
+            Time delay (in seconds). Maximum time delay is 59 seconds.
+
+        Notes
+        -----
+        The command following the :ref:`wait` will not be processed until the specified wait time increment
+        has elapsed. Useful when reading from a prepared input file to cause a pause, for example, after a
+        display command so that the display can be reviewed for a period of time. Another "wait" feature is
+        available via the :ref:`ask` command.  This command is valid in any processor.
         """
         command = f"/WAIT,{dtime}"
         return self.run(command, **kwargs)
+
+    def cycle(self, **kwargs):
+        r"""Bypasses commands within a do-loop.
+
+        Mechanical APDL Command: `\*CYCLE <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_CYCLE.html>`_
+
+        Notes
+        -----
+        Bypasses all commands between this command and the :ref:`enddo` command within a do-loop. The next
+        loop (if applicable) is initiated. The cycle option may also be conditionally executed [Use the
+        :ref:`if` ]. The :ref:`cycle` command must appear on the same file as the :ref:`do` command.  This
+        command is valid in any processor.
+        """
+        command = f"*CYCLE"
+        return self.run(command, **kwargs)
+
+    def endif(self, **kwargs):
+        r"""Ends an if-then-else.
+
+        Mechanical APDL Command: `\*ENDIF <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_ENDIF.html>`_
+
+        Notes
```

Comment:
In the future I would like to add bits here.. like how these commands are compared to the python ones.